Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -0,0 +1,314 @@
+﻿using Kudu.Contracts.Tracing;
```

Comment:
Please add a unit test class for ProfileManager that exercises and tests all its methods and logic